src/main/java/au/gov/geoscience/portal/server/controllers/RemanentAnomaliesController.java

```diff
@@ -0,0 +1,42 @@
+package au.gov.geoscience.portal.server.controllers;
+
+import org.auscope.portal.core.server.controllers.BasePortalController;
+import org.auscope.portal.core.util.FileIOUtil;
+import org.auscope.portal.core.util.SLDLoader;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.ByteArrayInputStream;
+import java.io.OutputStream;
+
+@Controller
+public class RemanentAnomaliesController extends BasePortalController {
+
+    public RemanentAnomaliesController() {
+    }
+
+    @RequestMapping("/remanentAnomaliesFilterStyle.do")
+    public void remanentAnomaliesFilterStyle(@RequestParam(required = false, value = "serviceUrl") String serviceUrl,
+                                           @RequestParam(required = false, value = "name") String name,
```

Comment:
Hi @geng-liang, just remiove all the unnecessary parameters from the controller call

Reply:
Unnecessary parameters have been removed from the controller.